csharp/ql/test/library-tests/frameworks/EntityFramework/EntityFrameworkCore.cs

```diff
@@ -1,38 +1,66 @@
 using Microsoft.EntityFrameworkCore;
 using System;
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations.Schema;
-using System.Data.Common;
+using System.Linq;
 
 namespace EFCoreTests
 {
     class Person
     {
-      public int Id { get; set; }
-      public string Name { get; set; }
-    
-      [NotMapped]
-      public int Age { get; set; }
+        public virtual int Id { get; set; }
+        public virtual string Name { get; set; }
+
+        [NotMapped]
+        public string Title { get; set; }
+
+        // Navigation property
+        public ICollection<Address> Addresses { get; set; }
+    }
+
+    class Address
+    {
+        public int Id { get; set; }
+        public string Street { get; set; }
```

Comment:
Good question! This is because of [the access path length restriction](https://github.com/github/codeql/blob/main/csharp/ql/src/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll#L22).